### DISCUSSION TITLE
v9 Nav component ETA

### DISCUSSION THREAD

**Discussion by blendsdk** [microsoft/fluentui#32146] [Q&A] [answered]
Hi,

Could you please provide an ETA regarding the Nav component? It is in preview at the moment and I would like to know how much you expect it to change? I would like to use it in a production setting.

Thanks.

**Comment by mltejera** [comment, marked as answer]
Hey @blendsdk - Thanks so much for reaching out, and glad you're interested in the component!

We need to get some feedback, iterate and validate with at least 2 internal teams before we mark the API as stable. Given the large nature of this component, I expect this process to take 6-12 months before we're comfortable marking the API as stable and commit to not making any breaking changes.

**Reply by blendsdk** [reply]
Hi @mltejera thank you for the quick response. 

I have started using this component in a mini ERP system we have built in v8 and now migrating to v9. I will send you feedback as much as I can. 

Cheers.

**Reply by AkiraVoid** [reply]
@mltejera How does the "6-12 months" count? From the time the component gets published as a public review or from the time this comment was made?
